saphana-plugin/docs/SapHana-batchsink.md

```diff
@@ -0,0 +1,46 @@
+# SAP HANA Batch Sink
+
+
+Description
+-----------
+
+This sink is used whenever you need to write to a SAP HANA table.
+Suppose you periodically build a recommendation model for products on your online store.
+The model is stored in a FileSet and you want to export the contents
+of the FileSet to a SAP HANA table where it can be served to your users.
+
+Column names would be autodetected from input schema.
+
+
+Use Case
+--------
+This sink is used whenever you need to write to a SAP HANA table.
+Suppose you periodically build a recommendation model for products on your online store.
+The model is stored in a FileSet and you want to export the contents
```

Comment:
FileSet is a deprecated dataset type.

Reply:
Maybe this need to be separately, as other plugins have similar usages in documentation